src/platform/user/profile/vap-svc/util/local-vapsvc.js

```diff
@@ -167,7 +175,7 @@ export default {
     });
   },
   createTransactionFailure() {
-    return asyncReturn(
+    return asyncReject(
```

Comment:
Previously, when using the `asyncReturn` helper, [this line](https://github.com/department-of-veterans-affairs/vets-website/blob/d1a201f951cb2c7026f0b4adfa73ae104e515e19/src/platform/user/profile/vap-svc/actions/transactions.js#L235-L236) would not throw an error in the `try` block and instead the try block would continue running as if the response was a 200.